fele-client-service/restful-api/handlers/fele-client.handler.js

```diff
@@ -0,0 +1,60 @@
+const { createNetwork, deleteNetwork } = require('../../client-api/scripts/network')
+const { createChannel } = require('../../client-api/scripts/channel')
+
+const createNetworkHandler = async(req, res) => {
+    const networkName = req.query.networkName
+    await createNetwork(JSON.stringify(req.body), networkName)
+    res.status(201).send({
+        message: "Network Created Successfully", 
+        networkConfig: {...req.body}
+    })
+}
+ 
+
+const createChannelHandler = async (req, res) => {
+    const {networkName, channelName} = req.body
```

Comment:
can we destructure channelConfig also here to use in the createChannel.

Reply:
sure